be/src/storage/meta_reader.cpp

```diff
@@ -36,7 +36,9 @@ Status SegmentMetaCollecter::parse_field_and_colname(const std::string& item, st
 
 MetaReader::MetaReader() : _is_init(false), _has_more(false) {}
 
-MetaReader::~MetaReader() {}
+MetaReader::~MetaReader() {
+    Rowset::release_readers(_rowsets);
```

Comment:
If _rowsets are not aquired, this will operation will make mistake.

Reply:
It does happen, I will fix it.